lib/config/validation.ts

```diff
@@ -95,6 +95,8 @@ function getDeprecationMessage(option: string): string | undefined {
     branchName: `Direct editing of branchName is now deprecated. Please edit branchPrefix, additionalBranchPrefix, or branchTopic instead`,
     commitMessage: `Direct editing of commitMessage is now deprecated. Please edit commitMessage's subcomponents instead.`,
     prTitle: `Direct editing of prTitle is now deprecated. Please edit commitMessage subcomponents instead as they will be passed through to prTitle.`,
+    logFile: `Using logFile to specify log file name is deprecated now. Please use the enviroment variable LOG_FILE instead`,
+    logFileLevel: `Using logFileLevel to specify log level for file logging is deprecated now. Please use the enviroment variable LOG_FILE_LEVEL instead`,
```

Comment:
Do we print both these messages and the deprecation ones? Are they printed for global config too?

Reply:
These are the depreciation msgs which will be shown in the logs. If you mean to ask whether there will be 2 log messages, the answer is depends. 
If the logFile / logFileContext are of wrong types. Then there willl be two warning logs, one for depriciation and another for type mismatch else onlt the depreciation logs.